Cargo.toml

```diff
@@ -58,6 +56,8 @@ members = [
   "crates/diagnostic_tests",
 ]
 
+resolver = "2"
```

Comment:
i'm curious, is this a semantic change?

Reply:
No, functional. The position it was moved to was invalid and giving warnings